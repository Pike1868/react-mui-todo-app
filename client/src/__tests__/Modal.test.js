import React from "react";
import Modal from "../components/Modal";
import { render } from "@testing-library/react";

//Smoke Test
it("renders without crashing", () => {
  render(<Modal />);
});

//Snapshot Test
it("matches snapshot", () => {
  const { asFragment } = render(<Modal />);
  expect(asFragment()).toMatchSnapshot();
});
