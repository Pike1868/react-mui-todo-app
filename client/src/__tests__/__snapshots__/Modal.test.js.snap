// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="overlay"
  >
    <div
      class="modal"
    >
      <div
        class="form-title-container"
      >
        <h3>
          Let's  your task
        </h3>
        <button>
          X
        </button>
      </div>
      <form>
        <input
          maxlength="30"
          name="title"
          placeholder="Your task goes here"
          required=""
          value=""
        />
        <br />
        <label
          for="range"
        >
          Drag to select your current progress
        </label>
        <input
          id="range"
          max="100"
          min="0"
          name="progress"
          required=""
          type="range"
          value="0"
        />
        <input
          type="submit"
        />
      </form>
    </div>
  </div>
</DocumentFragment>
`;
